pages/android/dumpsys.md

```diff
@@ -0,0 +1,28 @@
+# dumpsys
+
+> Provide information about Android system services.
+> More information: <https://developer.android.com/studio/command-line/dumpsys>.
+
+- Get diagnostic output for all system services:
+
+`dumpsys`
+
+- Get diagnostic output for a specific system service:
+
+`dumpsys {{service}}`
+
+- List all services `dumpsys` can give information about:
+
+`dumpsys -l`
+
+- List service-specific arguments for a service:
+
+`dumpsys {{service}} -h`
+
+- Exclude a specific service from the diagnostic output:
+
+`dumpsys --skip {{service}}`
+
+- Specify a timeout period in seconds (defaults to 10s):
+
+`dumpsys -t {{timeout}}`
```

Comment:
I have no idea, what this example does, I just took the description from the Android page. I hope it's correct.

Reply:
No long option?

Reply:
At least not according to the Android page

Reply:
I test it but I don't understand what this command does:
```suggestion
- Specify a timeout period in seconds (defaults to 10s):

`dumpsys -t {{seconds}}`
```